docs/api/remote-procedure-calls-wallet.md

```diff
@@ -970,7 +970,7 @@ _Result---a description of the transaction_
 | → →<br>`fee`                 | number (dash)   | Optional<br>(0 or 1)        | If sending payment, the fee paid as a negative dash value.  May be `0`.  Not returned if receiving payment                                                                                                                                                                                                                                                                                    |
 | → →<br>`abandoned`           | bool            | Optional<br>(0 or 1)        | _Added in Bitcoin Core 0.12.1_<br><br>Indicates if a transaction is was abandoned:<br>• `true` if it was abandoned (inputs are respendable)<br>• `false`  if it was not abandoned<br>Only returned by _send_ category payments                                                                                                                                                                |
 | →<br>`hex`                   | string (hex)    | Required<br>(exactly 1)     | The transaction in serialized transaction format                                                                                                                                                                                                                                                                                                                                              |
-| →<br>`decoded`               | object          | Optional<br>(0 or 1)        | the decoded transaction (only present when `verbose` is passed), equivalent to the`verbose` is passed), equivalent to the RPC decoderawtransaction method, or the RPC getrawtransaction method when `verbose` is passed.                                                                                                                                                                                                                                                                                                                                             |
+| →<br>`decoded`               | object          | Optional<br>(0 or 1)        | The decoded transaction (only present when `verbose` is passed), equivalent to the RPC [`decoderawtransaction` method] (../api/remote-procedure-calls-raw-transactions.html#decoderawtransaction), or the RPC [`getrawtransaction` method] (../api/remote-procedure-calls-raw-transactions.html#getrawtransaction)when `verbose` is passed.                                                                                                                                                                                                                  |
```

Comment:
Ditto. Also the markdown link can't contain a space between the link text and the link target.
```suggestion
| →<br>`decoded`               | object          | Optional<br>(0 or 1)        | The decoded transaction (only present when `verbose` is passed), equivalent to the RPC [`decoderawtransaction` method](../api/remote-procedure-calls-raw-transactions.md#decoderawtransaction), or the RPC [`getrawtransaction` method](../api/remote-procedure-calls-raw-transactions.md#getrawtransaction) when `verbose` is passed.                                                                                                                                                                                                                  |
```